src/main.cpp

```diff
@@ -5795,9 +6025,10 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
         BOOST_FOREACH(CTransaction tx, thinBlock.vMissingTx) 
             mapMissingTx[tx.GetHash()] = tx;
 
-        LOCK2(cs_main, cs_xval);
         int missingCount = 0;
         int unnecessaryCount = 0;
+        {
+        LOCK2(cs_main, cs_xval);
```

Comment:
this is a matter of style..we see this throughout Bitcoin..,when locking we don't always indent.  I don't really mind either way but sometimes it's easier to read the code or is less code change.